k8s/gvclient.go

```diff
@@ -14,6 +15,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/codes"
+	errors2 "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
nit: wdyt about renaming this to k8serrors? I think `errors2` doesn't give enough context when someone notices its usage in this file

Reply:
Yeah, probably a good idea, that was goland's auto-choice